containerz/containerz.proto

```diff
@@ -53,28 +61,33 @@ service Containerz {
   // is returned. If the specified container image does not exist, this
   // operation is a no-op.
   rpc RemoveImage(RemoveImageRequest)
-    returns (RemoveImageResponse) {}
+      returns (RemoveImageResponse) {}
 
   // RemoveContainer deletes the container that matches the instance
   // name. If the specified container does not exist, this
   // operation is a no-op. The container should not be removed if it is
   // running unless force is specified in the request.
+  // The request must return an error with status code NotFound if the container
+  // is not found. If the container is running and the request did not set the
+  // force field to true then the request must return an error with status code
+  // FailedPrecondition.
   rpc RemoveContainer(RemoveContainerRequest)
-    returns (RemoveContainerResponse) {}
+      returns (RemoveContainerResponse) {}
 
   // ListContainer returns all containers that match the spec defined in the
   // request.
   rpc ListContainer(ListContainerRequest)
-    returns (stream ListContainerResponse) {}
+      returns (stream ListContainerResponse) {}
 
   // Start starts a container. If the image does not exist on the target,
   // Start returns an error. A started container is identified by an instance
   // name, which  can optionally be supplied by the caller otherwise the target
-  // should provide one. When the instance name already exists, the target 
-  // should return an error if the container is running or if it is stopped but 
-  // the request has more fields than just the instance name. If the container 
-  // is stopped and the request contains only the instance name then Start 
-  // should start the container
+  // should provide one. When the instance name already exists, the target
+  // should return an error if the container is running or if it is stopped but
```

Comment:
I think you lost a change to specify the error type here.

Reply:
Ah yes thank you.